ime/app/src/main/res/values/settings_defaults_dont_translate.xml

```diff
@@ -3,6 +3,7 @@
     <!-- DO NOT TRANSLATE -->
     <!-- Default settings values -->
     <string name="settings_default_default_domain_text">.com</string>
+    <bool name="settings_default_use_system_vibration">false</bool>
```

Comment:
I assume that we want this to be `true` for API29+? If so, you can override the value for `values-v29`. (similar to what we do with [API 30](https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/ime/app/src/main/res/values-v30/settings_defaults_dont_translate.xml)

Reply:
I actually meant for this to be `false` by default because support for the effects [is not mandatory](https://developer.android.com/reference/android/os/Vibrator#areAllEffectsSupported(int...)), but I looked more carefully and [it does fall back to a sane default](https://developer.android.com/reference/android/os/VibrationEffect#createPredefined(int)) if they are not supported.

I enabled it by default for 29+ and discovered via the tests that it wasn't correctly responding to night mode / low power, so I fixed that as well.

Definitely worth it!